src/back-end/lib/db.ts

```diff
@@ -1267,8 +1267,8 @@ async function rawCWUProposalSlimToCWUProposalSlim(connection: Connection, raw:
   };
 }
 
-export const readManyCWUProposals = tryDb<[Id], CWUProposalSlim[]>(async (connection, id) => {
-  const results = await connection<RawCWUProposalSlim>('cwuProposals as prop')
+export const readManyCWUProposals = tryDb<[Session, Id], CWUProposalSlim[]>(async (connection, session, id) => {
```

Comment:
Can you change `Session` to `AuthenticatedSession` since anon users won't be reading proposals?

Reply:
updated